jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -752,11 +745,11 @@ ecma_string_get_heap_number_size (mem_cpointer_t number_cp) /**< Compressed poin
  *         NULL - otherwise
  */
 const lit_utf8_byte_t *
-ecma_string_raw_chars (const ecma_string_t *string_p, /* ecma-string */
-                       lit_utf8_size_t *size_p, /* [out] size of the ecma string */
-                       bool *is_ascii_p) /* [out] true, if the string is an ascii
-                                          *             character sequence (size == length)
-                                          *       false, otherwise */
+ecma_string_raw_chars (const ecma_string_t *string_p, /**< ecma-string */
+                       lit_utf8_size_t *size_p, /**< [out] size of the ecma string */
+                       bool *is_ascii_p) /**< [out] true, if the string is an ascii
+                                          *               character sequence (size == length)
+                                          *         false, otherwise */
```

Comment:
You still have these changes here, although you've requested them to be moved to #997